nats/src/test/kotlin/rys/ajaxpetproject/nats/NatsControllersIT.kt

```diff
@@ -31,23 +40,26 @@ import rys.ajaxpetproject.request.update.create.proto.ChatUpdateRequest
 import rys.ajaxpetproject.request.update.create.proto.ChatUpdateResponse
 import rys.ajaxpetproject.service.ChatService
 import rys.ajaxpetproject.service.impl.ChatServiceImpl
+import rys.ajaxpetproject.service.impl.MessageServiceImpl
+import rys.ajaxpetproject.service.impl.UserServiceImpl
 import rys.ajaxpetproject.subjects.ChatSubjectsV1
 import java.time.Duration
 
 @SpringBootTest(classes = [NatsTestConfiguration::class])
 @ContextConfiguration(
     classes = [
-        ChatRepository::class, ChatServiceImpl::class,
+        ChatRepositoryImpl::class, ChatServiceImpl::class,
         NatsChatCreationController::class,
         NatsChatDeleteController::class,
-        NatsChatFindAllController::class,
-        NatsChatFindOneController::class,
-        NatsChatUpdateController::class,
-        NatsControllerConfigurerPostProcessor::class
-        ]
+        NatsChatFindAllController::class, MessageRepository::class, UserRepository::class,
+        NatsChatFindOneController::class, MessageServiceImpl::class, UserServiceImpl::class,
+        NatsChatUpdateController::class, SecurityConfiguration::class,
```

Comment:
maybe, you want to define a `@ComponentScan` class, that would have package scan.

```kotlin
@ComponentScan(value = ["rys.ajaxpetproject.nats"])
class NatsControllersScan
```

Reply:
Isn't it bad practice to scan whole package?

Reply:
I mean, it's tests. 
To many classes to scan looks bad - yes, but it better then sooner or later run Test with many unnecessary dependencies

Reply:
> Isn't it bad practice to scan whole package?

Why it's bad practice? 
In some cases it could be bad idea, but it depends.

What I see now, you enumerate all NATS controllers that is definitely worse than just pointinig package with NATS controllers to scan